lib/platform/azure/index.ts

```diff
@@ -55,19 +55,33 @@ const defaults: any = {
   hostType: PLATFORM_TYPE_AZURE,
 };
 
-export function initPlatform({
-  endpoint,
-  token,
-}: RenovateConfig): Promise<PlatformConfig> {
-  if (!endpoint) {
+export function initPlatform(
+  renovateConfig: RenovateConfig
+): Promise<PlatformConfig> {
+  if (!renovateConfig.endpoint) {
```

Comment:
Do not change this, only add required properties
```suggestion
export function initPlatform({
  endpoint,
  token,
  username,
  password,
}: RenovateConfig): Promise<PlatformConfig> {
  if (!endpoint) {
```